eng/common/TestResources/New-TestResources.ps1

```diff
@@ -101,6 +108,16 @@ function Retry([scriptblock] $Action, [int] $Attempts = 5) {
     }
 }
 
+function MergeHashes([hashtable] $source, [psvariable] $dest) {
+    foreach ($key in $source.Keys) {
+        if ($dest.Value[$key]) {
+            Write-Warning ("Overwriting '$($dest.Name).$($key)' with value '$($dest.Value[$key])' " +
```

Comment:
We should use our devops logging functions here. It doesn't have to happen in this PR but at some point we should switch to using those.

Reply:
How about I do a follow-on that changes them all? That way it's consistent across the script.

Reply:
I recommend a `Warn` function like I did for logging that 1) always calls Write-Warning, and 2) if running under AzDO (or `$CI` is enabled, which would be consistent) uses the AzDO logging. Remember that we devs use this too and `##vsts.something` isn't really going to stick out.

Reply:
https://github.com/Azure/azure-sdk-tools/blob/master/eng/common/scripts/logging.ps1#L6